lib/dal/src/pkg/import.rs

```diff
@@ -94,28 +92,55 @@ async fn import_change_set(
     Vec<bool /*ImportEdgeSkip*/>,
 )> {
     // let default_change_set_id = ctx.get_workspace_default_change_set_id().await?;
+
+    // Cache the intrinsic funcs pkg in case we need it.
+    let unsafe_to_install_intrinsic_funcs_pkg = SiPkg::load_from_spec(IntrinsicFunc::pkg_spec()?)?;
```

Comment:
Do not... install this... just use the func specs. Turns out, you need a full package to get `SiPkgFunc`... so I brought a whole package with me.